.github/workflows/cd.yml

```diff
@@ -49,11 +49,37 @@ jobs:
 
       - name: Publish Docs to Cloudflare Pages
         if: ${{ steps.release.outputs.releases_created }}
-        uses: cloudflare/pages-action@v1
+        uses: cloudflare/wrangler-action@v3
         with:
+          gitHubToken: ${{ secrets.GITHUB_TOKEN }}
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: 2238a825c5aca59233eab1f221f7aefb
+          command: pages deploy ./docs/build --project-name=sprinter-docs
+
+      - name: Publish POC to Cloudflare Pages
+        if: ${{ steps.release.outputs.releases_created }}
+        uses: cloudflare/wrangler-action@v3
+        with:
+          gitHubToken: ${{ secrets.GITHUB_TOKEN }}
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: 2238a825c5aca59233eab1f221f7aefb
+          command: pages deploy ./web/.svelte-kit/cloudflare --project-name=sprinter-poc
```

Comment:
dns needs to be changed and set to new project on cloudflare before this is merged and released no?

Reply:
It is set in a stupid way deploying on both things, so when DNS changes will be the same